File: dev-docs/snippets/documents/aliases-list-for-document.ts
import type { JsBaoClient } from "js-bao-wss-client";

// List all aliases pointing at a document.
export async function aliasesListForDocument(
  client: JsBaoClient,
  documentId: string,
) {
  // #region example
  const aliases = await client.documents.aliases.listForDocument(documentId);
  // #endregion example
  return aliases;
}

File: dev-docs/snippets/documents/aliases-list-for-document.swift
import JsBaoClient

// List all aliases pointing at a document as typed `[DocumentAliasInfo]`.
func aliasesListForDocument(client: JsBaoClient, documentId: String) async throws {
  // #region example
  let aliases = try await client.documents.aliases.listForDocument(
    documentId: documentId
  )
  let firstKey = aliases.first?.aliasKey
  // #endregion example
  _ = firstKey
}
